docker/env.yaml

```diff
@@ -102,15 +111,19 @@ dependencies:
   - pyrsistent
   - pystac
   - pystac-client
+  - pytest
   - python-dateutil
   - python-rapidjson
   - python-slugify
   - pytz
   - PyWavelets
   - PyYAML
+  - rasterstats
   - regex
   - requests
+  - request-aws4auth
   - rsa
+  - rsgislib
```

Comment:
Are these additional packages coming from the NCI environment? We may not need `rsgislib`; we've kind of moved away from using it, and in the past it's been really hard to install

Reply:
Yeah, I all additional packages were in the nci conda environment but not in the sandbox env, I imagined some would probably be unnecessary but I wasn't sure which

Reply:
Yep cool. I'd keep "rasterstats", "pytest", "geopandas" and "ipyleaflet" (suprised they're not already in the image), and possibly "mpi4py" (I vaguely remember Chad asking for that one).

I'd be tempted to drop "rsgislib", "glue-geospatial", and "glueviz".

I don't know enough about "checksumdir", "ephem", "datadog", "icu" and "request-aws4auth" to know if they're important... might ask Teams I think

Reply:
`datadog` might be required by `odc-tools`, but really should be fixed so that it isn't. It's for sending Stack Traces to a [Sentry](https://sentry.io/) server.

`checksumdir` is used by `eo-datasets3`, so is needed.

`requests-aws4auth` is probably a good idea, but I'm not sure what is using it.

`ephem` is for astronomy computations, it might be used for calculating solar angles and that sort of thing, for the ARD processing, but again, not sure.

`icu` is for handling unicode/globalisation. There's probably something that depends on it, it's quite a common dep.

I'm happy with Robbi's suggestion of dropping `rsgislib` and `glue-*`.

Reply:
Yeah, `datadog` is used in odc-tools [here](https://github.com/opendatacube/odc-tools/blob/5e086527eb7f93ed484e76859a50f246007dd850/apps/dc_tools/odc/apps/dc_tools/utils.py#L4)